pkg/scheduler/bind_plugin_test.go

```diff
@@ -65,7 +66,12 @@ func TestBindPluginSchedulerNameSet(t *testing.T) {
 			bindDoneChan := make(chan struct{})
 			// run the binding step in parallel so the created PR can be inspected and updated
 			go func() {
-				binder := &BindPlugin{client: client}
+				binder := &BindPlugin{
+					client: client,
+					config: &config.PlacementRequestBinderArgs{
+						Timeout: &metav1.Duration{Duration: 10 * time.Second},
```

Comment:
Would it work better to have the timeout set to what is considered a default? At least for the purpose of unit tests unless a different value is needed. Or, do those 10 seconds change the way the unit test work?

Reply:
done.